pegasus/routes/v2_section_routes.rb

```diff
@@ -21,7 +23,14 @@
 end
 
 # Get the set of sections that the current user is enrolled in.
+# DEPRECATED: Use GET /dashboardapi/sections/membership instead
 get '/v2/sections/membership' do
+  # Notify Honeybadger to determine if this endpoint is still used anywhere
+  Honeybadger.notify(
+    error_class: "DeprecatedEndpointWarning",
+    error_message: 'Deprecated endpoint /v2/sections/membership called unexpectedly',
+  )
```

Comment:
great call on adding a honeybadger error in this case. Just a heads up, I would expect to see a bunch of traffic to this endpoint right away, due to the delay between when we update our server code and when any caching of js files expires and then users get around to refreshing their browser to pick up the new js files.